src/test/java/bio/terra/janitor/integration/common/configuration/IntegrationTestConfiguration.java

```diff
@@ -8,8 +10,8 @@
 /** Configs in integration test. */
 @Component
 @EnableConfigurationProperties
-@ConfigurationProperties(prefix = "test")
-public class TestConfiguration {
+@ConfigurationProperties(prefix = "it")
```

Comment:
"it" is not a very descriptive prefix. Prefer avoiding abbreviations.

Reply:
I thought it means for 'integrationTest' and this is how WM names it:
https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/test/resources/application-integration-test.properties#L1

Any other recommandations?full name like: integrationTest?

Reply:
yeah, "integrationTest" full name.